cmd/lncli/commands.go

```diff
@@ -1787,8 +1787,16 @@ var getChanInfoCommand = cli.Command{
 	ArgsUsage: "chan_id",
 	Flags: []cli.Flag{
 		cli.Uint64Flag{
-			Name:  "chan_id",
-			Usage: "the 8-byte compact channel ID to query for",
+			Name: "chan_id",
+			Usage: "the 8-byte compact channel ID to query for." +
```

Comment:
nit: fully capitalize first sentence as well or use semicolon instead of full stop to split the two sentences.
Also, missing space at end of first line. Both goes for the flag below as well.